nginx/default.template

```diff
@@ -12,6 +12,8 @@ server {
 
   location / {
     proxy_pass http://web;
+    client_max_body_size 20M
+    ;
```

Comment:
Any reason why there's a line break before this semicolon?

Reply:
Yes -- clumsiness